app/lib/request.rb

```diff
@@ -41,6 +42,12 @@ def headers
 
   private
 
+  def sanitize_url
+    if PrivateAddressCheck.resolves_to_private_address? @url.host
```

Comment:
`raise ... if ...` in one line

Reply:
And since the method is supposed to raise an exception, I would call it `sanitize_url!` with exclamation mark.